src/components/InputBook.js

```diff
@@ -1,25 +1,46 @@
-import React from 'react';
+import { React, useState } from 'react';
 import './InputBook.css';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { add } from '../redux/books/books';
 
-const InputBook = () => (
-  <>
-    <h2 className="textheading">ADD NEW BOOK</h2>
-    <form className="form-container">
-      <input
-        type="text"
-        className="input-title"
-        placeholder="Title"
-        name="title"
-      />
-      <input
-        type="text"
-        className="input-title"
-        placeholder="Author"
-        name="title"
-      />
-      <button type="submit" className="input-submit">Submit</button>
-    </form>
-  </>
-);
+const InputBook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
```

Comment:
Please ensure that your ` InputBook.js` has a local state with  `title` and `author` according to the project requirement.